src/details/ArborX_DetailsBoundingVolumeHierarchyImpl.hpp

```diff
@@ -115,6 +119,7 @@ void BoundingVolumeHierarchyImpl<DeviceType>::queryDispatch(
     reallocWithoutInitializing( indices, n_results );
     int const invalid_index = -1;
     Kokkos::deep_copy( indices, invalid_index );
```

Comment:
Does that allocating memory for `offset` and finding out how much memory to allocate for `indices` before filling with the array with invalid entries belong to another region?

Reply:
No region. The goal was not to have 100% coverage of everything with region, but rather cover interesting pieces.

Reply:
How were interesting pieces identified?